src/components/pages/parameter-or-variable.jsx

```diff
@@ -20,15 +21,39 @@ const ParameterOrVariablePage = React.createClass({
     currency: PropTypes.string.isRequired,
     location: locationShape.isRequired,
     params: PropTypes.shape({ name: PropTypes.string.isRequired }).isRequired, // URL params
-    parameters: PropTypes.arrayOf(AppPropTypes.parameterOrScale).isRequired,
+    parameters: PropTypes.objectOf(AppPropTypes.parameterOrScale).isRequired,
     variables: PropTypes.arrayOf(AppPropTypes.variable).isRequired,
   },
+  getInitialState() {
+    const variable = this.props.variables.find(variable => variable.name === this.props.params.name)
+    return {variable: variable, waitingForResponse: true}
```

Comment:
What would be the point ? Is this a common practise to initialise states properties to null in React ?